media-converter/src/audioconv.rs

```diff
@@ -510,15 +510,15 @@ impl AudioConvState {
             if let Ok(transcoded_size) = read_fozdb.entry_size(AUDIOCONV_FOZ_TAG_PTNADATA, hash) {
                 /* success */
                 let mut buf = vec![0u8; transcoded_size].into_boxed_slice();
-                if let Ok(_) = read_fozdb.read_entry(AUDIOCONV_FOZ_TAG_PTNADATA, hash, 0, &mut buf, fossilize::CRCCheck::WithoutCRC) {
+                if read_fozdb.read_entry(AUDIOCONV_FOZ_TAG_PTNADATA, hash, 0, &mut buf, fossilize::CRCCheck::WithoutCRC).is_ok() {
```

Comment:
I'm not sure if _ actually makes rust write to memory, but it's more clear like this anyways